src/instance.ts

```diff
@@ -249,6 +249,16 @@ class Instance extends common.GrpcServiceObject {
     return new Backup(this, backupId);
   }
 
+  copy_backup(backupId: string, sourceBackupId: string): Backup {
```

Comment:
I feel that the name of this function in combination with the (order of) arguments is slightly confusing. `copy_backup` sounds like I am creating a copy of a backup, and normally copy methods take the source as the first argument and the destination as the second. Also, other methods in this class use camel casing, so the name should then be `copyBackup`.
But I'm also wondering whether this function should not return a reference to a backup that the user then needs to call `backup.create()` on, for two reasons:
1. Consistency with other methods: The `createDatabase(..)` method in this class does not return a reference to a database that can be created, but instead executes a `CreateDatabase` RPC. The name `copy_backup` sounded to me like this function would also execute an RPC.
2. Is there any other conceivable use case than calling `backup.create()` after calling `copy_backup(..)`?